include/cantera/thermo/CoverageDependentSurfPhase.h

```diff
@@ -0,0 +1,476 @@
+/**
+ *  @file CoverageDependentSurfPhase.h
+ *  Header for a thermodynamics model of a coverage-dependent surface
+ *  phase derived from SurfPhase, applying adsorbate lateral interaction
+ *  correction factors to the SurfPhase thermodynamic properties.
+ *  (see \ref thermoprops and class
+ *  \link Cantera::CoverageDependentSurfPhase CoverageDependentSurfPhase\endlink).
+ */
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at https://cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_COVERAGEDEPENDENTSURFPHASE_H
+#define CT_COVERAGEDEPENDENTSURFPHASE_H
+
+#include "SurfPhase.h"
+#include "cantera/base/Array.h"
+
+namespace Cantera
+{
+
+//! A thermodynamic model for a coverage-dependent surface phase, applying adsorbate
+//! lateral interaction correction factors to the ideal surface phase properties.
+/*!
+ * The ideal surface phase assumes no lateral interaction among adsorbates.
+ * This coverage-dependent surface phase allows adding lateral interaction
+ * correction terms to the ideal surface phase (SurfPhase) thermodynamic properties
+ * so that more accurate adsorbate thermochemistry can be achieved.
+ *
+ * ## Coverage-dependent Thermodynamic Properties Formulations
+ *
+ * At a low-coverage limit, an adsorbate thermochemistry is the same as
+ * that of ideal species since there are no adsorbates in the vicinity to cause
+ * lateral interaction. Therefore, it is logical to set ideal species properties
+ * as the low-coverage limit properties and add lateral interaction terms, i.e.
+ * lateral interaction correction terms, to them as excess properties.
+ * Accordingly, standard state coverage-dependent enthalpy, entropy,
```

Comment:
A little bit of wordsmithing (specifically, we try to avoid the Latin abbreviation "i.e."):
```suggestion
 * lateral interaction. Therefore, it is logical to set ideal species properties
 * as the low-coverage limit and add lateral interaction correction terms to them as
 * excess properties. Accordingly, standard state coverage-dependent enthalpy, entropy,
```